content/docs/guides/browser-automation.md

```diff
@@ -15,9 +15,9 @@ Cucumber is not a browser automation tool, but it works well with the following
 
 WebDriver is designed to provide a simpler, more concise programming interface in addition to addressing some limitations in the Selenium-RC API. Selenium-WebDriver was developed to better support dynamic web pages where elements of a page may change without the page itself being reloaded. WebDriver's goal is to supply a well-designed object-oriented API that provides improved support for modern advanced web-app testing problems.
```

Comment:
... programming interface than some other tools. Selenium-WebDriver better supports ....

Reply:
Hi, well, seems this part is from legacy version of selenium. 
Let's keep this commit for URL replacing. 
Other parts should better to rewrite the document in the future.

Reply:
Yes lets fix this up.